src/shared/proto/messages_robocup_ssl_detection_tracked.proto

```diff
@@ -0,0 +1,106 @@
+syntax = "proto2";
+
+message Vector2 {
+    required float x = 1;
+    required float y = 2;
+}
+
+message Vector3 {
+    required float x = 1;
+    required float y = 2;
+    required float z = 3;
+}
+
+// The team color of the robot
+enum TeamColor {
+    UNKNOWN = 1;
+    YELLOW = 2;
+    BLUE = 3;
+}
+
+message RobotId {
+    // The robot number
+    required uint32 id = 1;
+    // The team color
+    required TeamColor team_color = 2;
+}
+
+// A single tracked ball
+message TrackedBall {
+    // The quality of the detection
+    // A value between 0 (not visible) and 1 (visible)
+    // The exact implementation depends on the source software
+    optional float quality = 1;
+
+    // The position (x, y, height) [m] in the ssl-vision coordinate system
+    required Vector3 pos = 2;
+
+    // The velocity [m/s] in the ssl-vision coordinate system
+    optional Vector3 vel = 3;
+}
+
+// A ball kicked by a robot, including predictions when the ball will come to a stop
+message KickedBall {
```

Comment:
As we at ER-Force will implement this protocol, I can already say that we currently do not have the required information to fill this message. While it may be produced as intermediate steps in the tracking computation, we do not propagate it to other parts of our framework. That is why we would likely have to omit this message.

Reply:
That's fine, the kicked_ball field is optional in the frame. But I added a note that explicitly says that some implementations will not set it at all.
Or could you set some of the fields in this message? In this case, are the optional and required attributes ok?

Reply:
Without changes to our internal messages, I could currently not send out any fields of this message. The position, velocity and timestamp is internal to the tracking and the stop position and time are not part of our tracking but computed in the typescript/lua strategy code, to which the planned CLI for this will have no access.
Both could be changed, but currently we only have the next touchdown position of the ball (when it starts the next bounce) and if it is currently bouncing as additional data.

Reply:
next touch down would indeed be interesting, I'll add this as well.

Reply:
added. Is it sufficient to encode "currently bouncing" as an empty list of touch down locations?
We provide all touch down locations, that's why I defined a list, but it's fine to just add one location.

Reply:
Should be fine, yes

Reply:
There seems to be some ambiguity in how to interpret this message as is: The touch down locations may 1) be absent entirely, 2) contain only the first touch down location, or 3) contain all subsequent locations (or some subset thereof). 
It would be good to have an enum to indicate which of these options apply. For example, using something like:
```
enum ChipKickDetectionSupport {
  UNSUPPORTED = 0;
  FIRST_TOUCHDOWN = 1;
  MULTIPLE_TOUCHDOWNS = 2;
}
```

Reply:
good point. I've added an Enum similar to your suggestion.